app/docs/setup_code.tsx

```diff
@@ -197,8 +202,32 @@ export default class SetupCodeComponent extends React.Component<Props, State> {
     event.target.innerText = "Copied!";
   }
 
+  fetchAPIKeyValue(bazelConfigResponse: bazel_config.IGetBazelConfigResponse, selectedIndex: number) {
+    const creds = bazelConfigResponse.credential;
+    if (!creds) {
+      return;
+    }
+
+    const selectedCreds = creds[selectedIndex];
+    if (selectedCreds.apiKey && !selectedCreds.apiKey.value) {
+      rpcService.service
+        .getApiKey(
+          api_key.GetApiKeyRequest.create({
+            apiKeyId: selectedCreds.apiKey.id,
+          })
+        )
+        .then((getApiKeyResp) => {
+          selectedCreds.apiKey!.value = getApiKeyResp.apiKey!.value;
+          this.setState({ bazelConfigResponse: this.state.bazelConfigResponse });
+        })
+        .catch((e) => error_service.handleError(e));
+    }
+  }
+
   onChangeCredential(e: React.ChangeEvent<HTMLSelectElement>) {
-    this.setState({ selectedCredentialIndex: Number(e.target.value) });
+    const selectedIndex = Number(e.target.value);
+    this.fetchAPIKeyValue(this.state.bazelConfigResponse!, selectedIndex);
+    this.setState({ selectedCredentialIndex: selectedIndex });
```

Comment:
We might need to implement a hide/show button for this page (Quickstart), otherwise, we will generate an Audit Log entry every time this page is loaded. 🤔 

Might not be within the scope of this PR though. WDYT?

Reply:
This seems fine for now. I could be wrong but I don't expect this page to be accessed very frequently. If we see any customer confusion wrt this we can revisit it.